Sources/Profile-Modify.php

```diff
@@ -642,6 +642,13 @@ function $preload:		A function that is used to load data required for this eleme
 			'size' => 50,
 			'permission' => 'profile_website',
 			'link_with' => 'website',
+			'input_validate' => function(&$value) use ($smcFunc)
+			{
+				if ($smcFunc['strlen']($value) > 250)
```

Comment:
This should use `mb_strlen($value)` rather than `$smcFunc['strlen']($value)`. This is because `$smcFunc['strlen']()` counts HTML entities like `&amp;` as single characters, but MySQL will count `&amp;` as five characters.